runelite-client/src/main/java/net/runelite/client/plugins/loginscreen/LoginScreenPlugin.java

```diff
@@ -326,4 +337,36 @@ private SpritePixels getFileSpritePixels(String file)
 
 		return null;
 	}
+
+	/**
+	 * Takes an image file and returns a SpritePixels object of the image data
+	 *
+	 * @param file A File object for the image
+	 * @return A SpritePixels object containing the image data
+	 */
+	private SpritePixels getFileSpritePixels(File file)
+	{
+		try
+		{
+			BufferedImage image;
+			synchronized (ImageIO.class)
+			{
+				image = ImageIO.read(file);
+			}
+
+			if (image.getHeight() > Constants.GAME_FIXED_HEIGHT)
+			{
+				final double scalar = Constants.GAME_FIXED_HEIGHT / (double) image.getHeight();
+				image = ImageUtil.resizeImage(image, (int) (image.getWidth() * scalar), Constants.GAME_FIXED_HEIGHT);
+			}
+
+			return ImageUtil.getImageSpritePixels(image, client);
+		}
+		catch (IOException | NullPointerException e)
```

Comment:
Catching `NullPointerException` is new compared to master's version of this code. Where would we expect to see that? I see `ImageIO.read()` would throw `IllegalArgumentException` if `file` was null, but that case isn't handled either?

Reply:
From my testing it threw `NullPointerException ` but I will do some more testing to confirm.

Reply:
I added this as `NullPointerException `is thrown when an invalid file is read (such as an .mp3 file renamed to .png). In the current code there is no way for `file` to be null but it would be worth adding a check for this in case the code is restructured in the future.

Reply:
This functionality has been added on latest commit.